pages/your_tasks/data_discoverability.md

```diff
@@ -0,0 +1,79 @@
+
+### Considerations
+
+* Detailed Metadata: ensure comprehensive metadata for your datasets, including detailed descriptions of studies, datasets, variables, and any available distributions.
+* Data Catalogs and Repositories: use well-maintained online catalogs and repositories that support controlled access data, and check for advanced search features to filter datasets by specific attributes.
+* Data Access Policies: get familiar with the data access policies of different repositories and datasets, understanding the requirements and procedures for requesting access to controlled data.
+* Ethical and Legal Compliance: ensure compliance with ethical guidelines and legal regulations governing data use and sharing, and obtain necessary approvals from institutional review boards or ethics committees if required. Check the [GDPR compliance](gpdr_compliance) and [Ethical aspects](ethics) pages for more information.
+* Data Access Request Process: be aware that the process for requesting and obtaining data access can be time-consuming, and prepare detailed justifications for data access requests, including research objectives and intended analyses.
+* Privacy and Security Measures: implement robust privacy and security measures to protect sensitive data during discovery and after access is granted, ensuring data handling practices comply with data protection regulations. Check the [Data sensitivity](data_sensitivity) page for more information.
+
+### Solutions
+
+{% tool "beacon" %}, developed through the Global Alliance for Genomics and Health (GA4GH) Discovery workstream, and with substantial support from ELIXIR, serves as a data discovery protocol and specification defining an open standard for discovering genomic and phenoclinic data in biomedical research and clinical applications.
+
+The latest version, {% tool "beacon-v2" %}, introduced expanded query options, enabling the retrieval of biological or technical (meta)data through filters defined via CURIEs. This includes, but not limited to, parameters such as phenotypes, disease codes, sex, or age, providing researchers with a nuanced approach to data inquiries.
+
+Beacon v2 is organised in two main blocks: the Beacon Framework and the Beacon Model. 
+The Framework defines the format for the requests and responses, whereas the Model defines the structure of the biological data response.
+
+This dual-system approach not only broadens the scope for diverse Models – using different domains such as images, pathogens, or infectious diseases – but also reinforces the adaptability of the Framework. The overall function of these components is to provide the instructions to design a REST API that could be implemented as a stand-alone product or, preferably, extending existing data management solutions. 
+
+Consequently, the 'beaconised' data represents a significant enhancement in data discoverability with minimal risks. 
+
+Currently, there are two ways to implement a Beacon:
```

Comment:
"Currently, there are two..." can be placed next to the previous sentence after "...minimal risks."